validation-test/stdlib/RuntimeSymbols.swift

```diff
@@ -1,6 +1,6 @@
 # Verify symbols from libswiftCore.dylib.
 
-UNSUPPORTED: OS=linux
+UNSUPPORTED: OS=linux-gnu
```

Comment:
I'm not sure that's actually an improvement. We'll never use it anywhere else. It's probably okay to just have two tests.